user_management/api/views.py

```diff
@@ -217,15 +218,25 @@ class VerifyAccountView(views.APIView):
     """
     permission_classes = [AllowAny]
     ok_message = _('Your account has been verified.')
+    # Default token never expires.
+    DEFAULT_VERIFY_ACCOUNT_EXPIRY = None
 
     def initial(self, request, *args, **kwargs):
         """
         Use `token` to allow one-time access to a view.
 
+        Token expiry can be set in `settings` with `VERIFY_ACCOUNT_EXPIRY` and is
+        set in seconds.
+
         Set user as a class attribute or raise an `InvalidExpiredToken`.
         """
         try:
-            email_data = signing.loads(kwargs['token'])
+            max_age = settings.VERIFY_ACCOUNT_EXPIRY
```

Comment:
I was thinking about the app config, maybe we should wait until we drop support for 1.6